Extensions/TweenBehavior/JsExtension.js

```diff
@@ -141,18 +142,72 @@ module.exports = {
         'JsPlatform/Extensions/tween_behavior24.png',
         'JsPlatform/Extensions/tween_behavior32.png'
       )
+      .setHidden()
       .addCodeOnlyParameter('currentScene', '')
       .addParameter('identifier', _('Tween Identifier'), 'sceneTween')
       .addParameter('scenevar', _('The variable to tween'), '', false)
       .addParameter('expression', _('Final value'), '', false)
-      .addParameter('expression', _('Duration'), '', false)
+      .addParameter('expression', _('Duration (in milliseconds)'), '', false)
       .addParameter('stringWithSelector', _('Easing'), easingChoices, false)
       .getCodeExtraInformation()
-      .setIncludeFile('Extensions/TweenBehavior/shifty.js')
-      .addIncludeFile('Extensions/TweenBehavior/shifty_setup.js')
+      .setIncludeFile('Extensions/TweenBehavior/standard-easing-functions.js')
       .addIncludeFile('Extensions/TweenBehavior/tweentools.js')
       .setFunctionName('gdjs.evtTools.tween.tweenVariableNumber2');
 
+    extension
+      .addAction(
+        'AddLayoutValueTween',
+        _('Add scene value tween'),
+        _('Add a tween animation for an scene value.'),
```

Comment:
`a scene value` 
Also, should this be `a scene variable` ?

Also, I'm not sure about the wording, should this just be "Tween a scene variable's value"? or "Animate a scene variable's value with tween"?

Reply:
The deprecated actions were using variables but it's not how GDevelop extension normally works. Values must be retrieved using expressions.
I gave more details in this PR description.